examples/atari/atari_dqn.py

```diff
@@ -104,7 +104,8 @@ def test_dqn(args: argparse.Namespace = get_args()) -> None:
     net = DQN(*args.state_shape, args.action_shape, args.device).to(args.device)
     optim = torch.optim.Adam(net.parameters(), lr=args.lr)
     # define policy
-    policy: DQNPolicy = DQNPolicy(
+    policy: DQNPolicy | ICMPolicy
```

Comment:
Ahh, it's necessary b/c of an open issue in mypy: https://github.com/python/mypy/issues/3737

Reply:
I didn't know about the issue in mypy, thx. 

I looked into our issue, whether it's b/c `policy` is defined multiple times with different policies, but mypy wants to type the `policy` even if it is defined once.

Btw, [in the overview tutorial](https://github.com/thu-ml/tianshou/blob/master/docs/02_notebooks/L0_overview.ipynb) `policy` is typed with `BasePolicy`. I think it should be typed with `PPOPolicy` there so that we get all the `PPOPolicy`-specific attrs and methods in the IDE.

Reply:
Right, would be better - could you fix it in the tutorial pls?

Reply:
Yes.

Reply:
Done.